### DISCUSSION TITLE
Publish official Godot docker images somewhere

### DISCUSSION THREAD

**Discussion by roookeee** [godotengine/godot-proposals#5853] [General Discussions]
**Description**
(Most) Developers want to build their Godot based software in some sort of CI environment. Using Docker for such tasks has become the industry standard. There are currently no (official) docker images for Godot so we either have to rely on third-party images (often from random people around the world or bound to specific CI pipelines like Github actions, which is non-optimal) or build our own images, especially for the current Godot 4 beta which seems to last for a little while.

**Proposal**
Automatically release official Godot images for every release of Godot, including the beta releases. These are built without any extensions for Linux only. Developers which are dependent on built-dependent changes to Godot will always need a custom build process. Built-in build switches (like the double precision one) could be considered in the future.

**Comment by Calinou** [comment]
This is already being tracked in https://github.com/godotengine/godot-proposals/issues/1411.
